### PR TITLE
Unified the args and attrbs of the message* classes.

### PR DESCRIPTION
**Pull request by RobiMez**
### Checklist for PRs

- [x] Added `.. versionadded:: version`, `.. versionchanged:: version` or `.. deprecated:: version` to the docstrings for user facing changes (for methods/class descriptions, arguments and attributes)
- [ ] Created new or adapted existing unit tests
- [ ] Documented code changes according to the [CSI standard](https://standards.mousepawmedia.com/en/stable/csi.html)
- [ ] Added myself alphabetically to `AUTHORS.rst` (optional)
- [ ] Added new classes & modules to the docs and all suitable `__all__` s

### CODE REVIEW

**Comment by RobiMez** [comment]
looking into what the sphinx error is

File: telegram/_message.py [hunk 1]
```diff
@@ -267,6 +269,8 @@ class Message(TelegramObject):
             or from anonymous administrators, information about the original sender chat.
         forward_from_message_id (:obj:`int`): Optional. For forwarded channel posts, identifier of
             the original message in the channel.
+        forward_sender_name (:obj:`str` :noindex: ): Optional. Sender's name for messages
+        forwarded from users who disallow adding a link to their account in forwarded messages.
```
Comment (devops117) [inline telegram/_message.py:273]: missing a whitespace?

Comment (RobiMez) [reply telegram/_message.py:273]: hmm , maybe a tab on line 273 ?
ill add one . white space tho, where ?

Comment (devops117) [reply telegram/_message.py]: yep, tab

Comment (RobiMez) [reply telegram/_message.py]: done.

**Comment by clot27** [comment]
Hi, thanks for the PR! Although the `Message()` class has already been unified in #3217, and I don't see the need to add duplicate `forward_sender_name` there 🤔

File: telegram/_message.py [hunk 2]
```diff
@@ -267,6 +269,8 @@ class Message(TelegramObject):
             or from anonymous administrators, information about the original sender chat.
         forward_from_message_id (:obj:`int`): Optional. For forwarded channel posts, identifier of
             the original message in the channel.
+        forward_sender_name (:obj:`str` :noindex: ): Optional. Sender's name for messages
+            forwarded from users who disallow adding a link to their account in forwarded messages.
```
Comment (harshil21) [inline telegram/_message.py:273]: no need of re-documenting this as clot pointed out

File: telegram/_messageentity.py [hunk 3]
```diff
@@ -63,13 +63,26 @@ class MessageEntity(TelegramObject):
...
+        type (:obj:`str`): Type of the entity. Can be :attr:`MENTION` (@username),
+            :attr:`HASHTAG`, :attr:`BOT_COMMAND`,
+            :attr:`URL`, :attr:`EMAIL`, :attr:`PHONE_NUMBER`, :attr:`BOLD` (bold text),
+            :attr:`ITALIC` (italic text), :attr:`STRIKETHROUGH`, :attr:`SPOILER` (spoiler message),
+            :attr:`CODE` (monowidth string), :attr:`PRE` (monowidth block), :attr:`TEXT_LINK` (for
+            clickable text URLs), :attr:`TEXT_MENTION` (for users without usernames),
+            :attr:`CUSTOM_EMOJI` (for inline custom emoji stickers).
+
+            .. versionadded:: 20.0
+                added inline custom emoji
```
Comment (harshil21) [inline telegram/_messageentity.py:75]:
```suggestion
                Added inline custom emoji
```
can you do this change in the Args: section also?

File: telegram/_messageentity.py [hunk 4]
```diff
@@ -63,13 +63,26 @@ class MessageEntity(TelegramObject):
...
         length (:obj:`int`): Length of the entity in UTF-16 code units.
-        url (:obj:`str`): Optional. Url that will be opened after user taps on the text.
+        url (:obj:`str`): Optional. For :attr:`TEXT_LINK` only, url that will be opened after
+            user taps on the text.
         user (:class:`telegram.User`): Optional. The mentioned user.
-        language (:obj:`str`): Optional. Programming language of the entity text.
-        custom_emoji_id (:obj:`str`): Optional. Unique identifier of the custom emoji.
+        language (:obj:`str`): Optional. For :attr:`PRE` only, The programming language of
+            the entity text.
+        custom_emoji_id (:obj:`str`): Optional.For :attr:`CUSTOM_EMOJI` only, unique identifier
```
Comment (harshil21) [inline telegram/_messageentity.py:83]:
```suggestion
        custom_emoji_id (:obj:`str`): Optional. For :attr:`CUSTOM_EMOJI` only, unique identifier
```

**Review by harshil21 — COMMENTED** [review]
nice catches!

File: telegram/_message.py (hunk 2 above)
Comment (RobiMez) [reply telegram/_message.py]: done in 
9fab68e49510ed8e623897941121f8742c80863e

**Review by harshil21 — APPROVED** [review]

**Pull request merged**